src/js/modal-dialog.js

```diff
@@ -256,6 +256,7 @@ class ModalDialog extends Component {
         let temp = this.contentEl_;
         this.contentEl_ = this.el_;
         close = this.addChild('closeButton');
+        close.controlText('Close Modal Dialog');
```

Comment:
I think this would be better like this:

```js
close = this.addChild('closeButton', {controlText: 'Close Modal Dialog'});
```

It avoids multiple internal calls to `controlText()`.